### ISSUE TITLE
Container exits due to `no-updates` option

### ISSUE THREAD

**Issue by Hudater** [linuxserver/docker-embystat#14]
[linuxserverurl]: https://linuxserver.io
[![linuxserver.io](https://raw.githubusercontent.com/linuxserver/docker-templates/master/linuxserver.io/img/linuxserver_medium.png)][linuxserverurl]

------------------------------

## Expected Behavior

The container should startup and serve on specified port

## Current Behavior

Container exits with error

## Steps to Reproduce

1. Launch container with latest image

## Environment
**OS:** Ubuntu Server 22.04 LTS
**CPU architecture:** x86_64
**How docker service was installed:** docker repo

## Command used to create docker container (run/create/compose/screenshot)

```yaml
---
version: "2.1"
services:
  embystat:
    image: linuxserver/embystat:latest
    container_name: embystat
    networks:
      - proxy
    environment:
      - PUID=${PUID}
      - PGID=${PGID}
      - TZ=${TZ}
    volumes:
      - "${CFG_DIR}/embystat:/config"
    ports:
      - 6555:6555
    restart: unless-stopped

networks:
  proxy:    # rename this to your custom docker network.
    external: true
```

## Docker logs

```
EmbyStat 0.2.0.0
Copyright (C) 2022 EmbyStat

:
  Option 'no-updates' is unknown.
  -p, --port               Set the port EmbyStat needs to be hosted on.
  -n, --disable-updates    Disable all update flow and pages on the server.
                           is <current-directory>
                           default is <current-directory>/logs
  -c, --config-dir         (Default: ) Folder where config files are stored,
  -u, --listen-urls        (Default: ) Set the url's where EmbyStat needs to
                           you want to define more then one url.
SYSTEM	Server shutdown
  --version                Display version information.
  -s, --service            Indicate EmbyStat is running as a service
  --help                   Display this help screen.
                           port number, it will be added automatically. Use ; if
                           listen on. Default is http://* but you can change it
                           to only loopback address here if needed. Don't add
                           2: Information
  -g, --log-level          Set the proper log level
                           1: Debug
                           default is <current-directory>
  -l, --log-dir            (Default: ) Folder where log files are stored,
  -d, --data-dir           (Default: ) Folder where database is stored, default
```

**Comment by thespad** [comment]
We are aware and this is being addressed by #13 but it requires feedback from the developer as the new flags are currently being ignored (or misreported).

**Comment by Hudater** [comment]
Closing issue
